README.md

```diff
@@ -506,6 +506,17 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Issues**: Check [Issues](https://github.com/search?q=org%3AZupIT+label%3A%22good+first+issue%22+state%3Aopen+type%3Aissue+user%3AZupIT)
 - **Notes**: Redeem your T-Shirt after ensuring 2 PRs at [Zup Website](https://insights.zup.com.br/hacktoberfest)
 
+ #### **The Original - DigitalOcean, Appwrite, Intel, and Deepsource** (T-shirts,stickers)
+- **Requirements**: 4 pull requests in any eligible repository
+- **Swag**: T-shirt, stickers
+- **How to sign up**: [Hacktoberfest Website](https://hacktoberfest.digitalocean.com/)
+- **Notes**: For your PR to count it must be:
+Submitted in a public repo, AND
+The PR is labelled as hacktoberfest-accepted by a maintainer, OR
+Submitted in a repo labelled hacktoberfest, AND
+Merged, OR
+Approved
+
```

Comment:
Hey @Avik17. The official Hacktoberfest swag is already in the Least Involvement to Most Involvment section of the Swag List under the 2+ PR section. The pull requests do not need to be merged to count for Hacktoberfest. If a repo maintainer uses the ```hacktoberfest-approved``` label, that's good enough. 👍